regression/global_C96_fv3aerorad.sh

```diff
@@ -0,0 +1,344 @@
+done
+
+# Copy observational data to $tmpdir
+$ncpl $datobs/${prefix_obs}.prepbufr                ./prepbufr
+$ncpl $datobs/${prefix_obs}.prepbufr.acft_profiles  ./prepbufr_profl
+$ncpl $datobs/${prefix_obs}.nsstbufr                ./nsstbufr
+$ncpl $datobs/${prefix_obs}.airsev.${suffix_obs}        ./airsbufr
+$ncpl $datobs/${prefix_obs}.mtiasi.${suffix_obs}        ./iasibufr
+$ncpl $datobs/${prefix_obs}.cris.${suffix_obs}          ./crisbufr
+
+# Copy bias correction, atmospheric and surface files
+$ncpl $datges/gfs.t18z.abias           ./satbias_in
+$ncpl $datges/gfs.t18z.abias_pc        ./satbias_pc
+$ncpl $datges/gfs.t18z.abias_air       ./aircftbias_in
+#$ncpl $datges/gfs.t18z.radstat         ./radstat.gdas
+
+if [[ "$endianness" = "Big_Endian" ]]; then
+   ln -s -f $datges/${prefix_sfc}.sfcf03            ./sfcf03
+   ln -s -f $datges/${prefix_sfc}.sfcf06            ./sfcf06
+   ln -s -f $datges/${prefix_sfc}.sfcf09            ./sfcf09
+elif [[ "$endianness" = "Little_Endian" ]]; then
+   ln -s -f $datges/${prefix_sfc}.sfcf03.le         ./sfcf03
+   ln -s -f $datges/${prefix_sfc}.sfcf06.le         ./sfcf06
+   ln -s -f $datges/${prefix_sfc}.sfcf09.le         ./sfcf09
+fi
+
+if [[ "$endianness" = "Big_Endian" ]]; then
+   ln -s -f $datges/${prefix_atm}.sigf03        ./sigf03
+   ln -s -f $datges/${prefix_atm}.sigf06        ./sigf06
+   ln -s -f $datges/${prefix_atm}.sigf09        ./sigf09
+elif [[ "$endianness" = "Little_Endian" ]]; then
+   ln -s -f $datges/${prefix_atm}.sigf03.le     ./sigf03
+   ln -s -f $datges/${prefix_atm}.sigf06.le     ./sigf06
+   ln -s -f $datges/${prefix_atm}.sigf09.le     ./sigf09
+fi
+
+if [[ "$endianness" = "Big_Endian" ]]; then
+   $ncpl $datges/${prefix_aer}.sigf03            ./aerf03
+   $ncpl $datges/${prefix_aer}.sigf06            ./aerf06
+   $ncpl $datges/${prefix_aer}.sigf09            ./aerf09
+elif [[ "$endianness" = "Little_Endian" ]]; then
+   $ncpl $datges/${prefix_aer}.sigf03.le         ./aerf03
+   $ncpl $datges/${prefix_aer}.sigf06.le         ./aerf06
+   $ncpl $datges/${prefix_aer}.sigf09.le         ./aerf09
+fi
+
+# Run GSI
+cd $tmpdir
+echo "run gsi now"
+eval "$APRUN $tmpdir/gsi.x > stdout 2>&1"
+rc=$?
+exit $rc
+
+
+# Loop over first and last outer loops to generate innovation
+# diagnostic files for indicated observation types (groups)
+#
+# NOTE:  Since we set miter=2 in GSI namelist SETUP, outer
+#        loop 03 will contain innovations with respect to 
+#        the analysis.  Creation of o-a innovation files
+#        is triggered by write_diag(3)=.true.  The setting
+#        write_diag(1)=.true. turns on creation of o-g
+#        innovation files.
+#
+
+
+echo "Time before diagnostic loop is `date` "
+cd $tmpdir
+loops="01 03"
+for loop in $loops; do
+
+case $loop in
+  01) string=ges;;
+  03) string=anl;;
+   *) string=$loop;;
+esac
+
+#  Collect diagnostic files for obs types (groups) below
+   listall="modis_aod_terra modis_aod_aqua"
```

Comment:
Is this correct? I thought diag_aero was set to false in the namelist? Do we need to save diag files for other obs types?

Reply:
I also feel it's not needed section, because the script exit after the GSI finished, line 291.
Since this script is based on global_C96_fv3aero.sh, so I kept the section about diagnostic files.